types/types.ts

```diff
@@ -1,12 +1,14 @@
+export type Role = 'staff' | 'patient'
```

Comment:
I think we should make role an enun where we map each role to the stringified version, this will reduce the need to use the actual string around our codebase and follows best practices

I'm not sure if typescript has this functionality but something like:
```
export type Role = enum {
staff : 'staff', 
patient : 'patient'
}

Reply:
That's a good call. For future reference the syntax is
```ts
export enum Role {
Patient="PATIENT",
Staff="STAFF"
}
```